Library/Homebrew/cmd/update.sh

```diff
@@ -288,12 +288,22 @@ EOS
   # check permissions
   if [[ "$HOMEBREW_PREFIX" = "/usr/local" && ! -w /usr/local ]]
   then
-    odie "/usr/local must be writable!"
+    odie <<-EOS
+/usr/local is not writable. You should probably change the
```

Comment:
Let's drop the probably; anyone who knows enough to disagree won't blindly follow the instruction.